src_testbed/testbed.rs

```diff
@@ -1552,6 +1552,7 @@ fn update_testbed(
     if state.running == RunMode::Step {
         state.running = RunMode::Stop;
     }
+    profiling::finish_frame!();
```

Comment:
`profiling::finish_frame!();` belongs here (in user code), as rapier isn't strictly the owner of the loop.